### PR TITLE
Start running tests against Python 3.5

### PR DESCRIPTION
**Pull request by dstufft**

### CODE REVIEW

File: .travis/run.sh [hunk 1]
```diff
@@ -24,5 +24,12 @@ else
         pyenv global pypy-2.6.0
     fi
 fi
+
+# If we're building on Python 3.5, then we need to use plain asserts until
+# pytest is updated.
```
Comment (alex) [inline .travis/run.sh]: Can you add a link to an issue here.

Comment (alex) [inline .travis/run.sh]: https://bitbucket.org/pytest-dev/pytest/issue/744/pytest-assert-rewriting-broken-with-python is the issue

**Comment by alex** [comment]
@dstufft URL chnaged here to https://github.com/pytest-dev/pytest/issues/744

**Comment by reaperhulk** [comment]
This was resolved in #2337

**Pull request closed without merging**